src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -297,6 +298,8 @@ def load_arguments(self, _):
         c.argument('workload_runtime', arg_type=get_enum_type(workload_runtimes), default=CONST_WORKLOAD_RUNTIME_OCI_CONTAINER)
         c.argument('enable_apiserver_vnet_integration', action='store_true', is_preview=True)
         c.argument('apiserver_subnet_id', validator=validate_apiserver_subnet_id, is_preview=True)
+        # no validation for aks create because it already only supports Linux.
+        c.argument('enable_custom_ca_trust', options_list=['--enable-custom-ca-trust'], action='store_true')
```

Comment:
No need for `options_list=['--enable-custom-ca-trust']` as this will be auto converted by cli.